proto/storage/cluster.proto

```diff
@@ -94,6 +95,53 @@ message AdmissionControllerConfig {
   bool enforce_on_updates = 5;
 }
 
+message ProcessConfig {
+  bool enabled = 1;
+}
+
+message NetworkConnectionConfig {
+  bool enabled = 1;
+  bool aggregate_external = 2;
+}
+
+message NetworkEndpointConfig {
+  bool enabled = 1;
+  bool include_listening_endpoint_processes = 2;
+}
+
+message CollectorNamespaceFeature {
+  oneof feature {
+    NetworkConnectionConfig network_connection = 1;
+  }
+}
+
+message CollectorNamespaceConfig {
+  message NamespaceRule {
+    string namespace = 1;
+    storage.MatchType match_type = 2;
+  }
+
+  CollectorNamespaceFeature feature = 1;
```

Comment:
I think this would make the feature embedded, instead of providing a reference. Is this intentional?

A config would look like this which reconfigures the feature in the `CollectorNamespaceConfig`, but I think it needs a reference to the feature config name?

```
collectorConfig:
  processConfig: ...
  networkConnectionConfig: 
      enabled: true
      aggregate_external: false
   collectorNamespaceConfig:
      - namespaceRule: 
          namespace: stackrox
          matchType: exact
         feature:
            network_connection:
                enabled: true
                aggregateExternal: false
```